crates/astria-bridge-contracts/src/lib.rs

```diff
@@ -698,9 +693,22 @@ struct EncodeMemoError {
     source: serde_json::Error,
 }
 
-fn memo_to_json<T: prost::Name + serde::Serialize>(memo: &T) -> Result<String, EncodeMemoError> {
-    serde_json::to_string(memo).map_err(|source| EncodeMemoError {
-        proto_message: T::full_name(),
+fn memo_to_json(
```

Comment:
The generic was not necessary because we only used this in one spot, right?

Also (because it no longer takes a memo and gives json, but actually creates the memo and encodes it as json):
```suggestion
fn make_withdrawal_memo(
```

Reply:
Done in [32b1522](https://github.com/astriaorg/astria/pull/1753/commits/32b15220e4e70a75cc134a150d1c0d3c9fdfd222).